package.json

```diff
@@ -30,7 +30,7 @@
     "jquery": "^3.6.0",
     "lato-font": "^3.0.0",
     "mini-css-extract-plugin": "^0.6.0",
-    "node-sass": "^4.13.1",
+    "node-sass": "^4.14.1",
```

Comment:
I haven't tested this one yet, so can't say what to look for other than build failures. Seems okay, but probably worth doing a thorough check on style output.